src/SFML.Graphics/Shape.cs

```diff
@@ -194,98 +189,86 @@ public Shape(Shape copy) :
         /// PointCount or GetPoint is different).
         /// </summary>
         ////////////////////////////////////////////////////////////
-        protected void Update()
-        {
-            sfShape_update(CPointer);
-        }
+        protected void Update() => sfShape_update(CPointer);
 
         ////////////////////////////////////////////////////////////
         /// <summary>
         /// Handle the destruction of the object
         /// </summary>
         /// <param name="disposing">Is the GC disposing the object, or is it an explicit call ?</param>
         ////////////////////////////////////////////////////////////
-        protected override void Destroy(bool disposing)
-        {
-            sfShape_destroy(CPointer);
-        }
+        protected override void Destroy(bool disposing) => sfShape_destroy(CPointer);
 
         ////////////////////////////////////////////////////////////
         /// <summary>
         /// Callback passed to the C API
         /// </summary>
         ////////////////////////////////////////////////////////////
-        private UIntPtr InternalGetPointCount(IntPtr userData)
-        {
-            return (UIntPtr)GetPointCount();
-        }
+        private UIntPtr InternalGetPointCount(IntPtr userData) => (UIntPtr)GetPointCount();
 
         ////////////////////////////////////////////////////////////
         /// <summary>
         /// Callback passed to the C API
         /// </summary>
         ////////////////////////////////////////////////////////////
-        private Vector2f InternalGetPoint(UIntPtr index, IntPtr userData)
-        {
-            return GetPoint((uint)index);
-        }
+        private Vector2f InternalGetPoint(UIntPtr index, IntPtr userData) => GetPoint((uint)index);
 
         [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
-        private delegate UIntPtr GetPointCountCallbackType(IntPtr UserData);
+        private delegate UIntPtr GetPointCountCallbackType(IntPtr userData);
 
         [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
-        private delegate Vector2f GetPointCallbackType(UIntPtr index, IntPtr UserData);
+        private delegate Vector2f GetPointCallbackType(UIntPtr index, IntPtr userData);
 
         private readonly GetPointCountCallbackType myGetPointCountCallback;
         private readonly GetPointCallbackType myGetPointCallback;
 
         private Texture myTexture = null;

```

Comment:
Can we add a rule to use `_`-prefix instead of `my`?

Reply:
Done.